.github/workflows/pull-release.yml

```diff
@@ -0,0 +1,137 @@
+
+      - name: Get latest charts for ingester
+        id: hypertrace-ingester
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/hypertrace-ingester
+   
+      - name: Get latest charts for ui
+        id: hypertrace-ui
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/hypertrace-ui
+
+      - name: Get latest charts for graphql
+        id: hypertrace-graphql
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/hypertrace-graphql
+
+      - name: Get latest charts for attribute-service
+        id: attribute-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/attribute-service
+
+      - name: Get latest charts for gateway-service
+        id: gateway-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/gateway-service
+
+      - name: Get latest charts for query-service
+        id: query-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/query-service
+
+      - name: Get latest charts for entity-service
+        id: entity-service
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: hypertrace/entity-service
+
+      - uses: actions/checkout@v2
+
+      - name: Update data services charts
+        run: ./.github/workflows/create-charts.sh data-services
+        env: 
+          kafka_version: ${{ steps.kafka.outputs.tag }}
+          pinot_version: ${{ steps.pinot.outputs.tag }}
+          schema_registry_version: ${{ steps.schema-registry.outputs.tag }}
+          mongodb_version: ${{ steps.mongodb.outputs.tag }}
+          zookeeper_version: ${{ steps.zookeeper.outputs.tag }}
+      
+      - name: Update platform services charts
+        run: ./.github/workflows/create-charts.sh platform-services
+        env: 
+          hypertrace_oc_collector_version : ${{ steps.opencensus-service.outputs.tag }}
+          hypertrace_ingester_version: ${{ steps.hypertrace-ingester.outputs.tag }}
+          hypertrace_ui_version: ${{ steps.hypertrace-ui.outputs.tag }}
+          hypertrace_graphql_version: ${{ steps.hypertrace-graphql.outputs.tag }}
+          attribute_service_version: ${{ steps.attribute-service.outputs.tag }}
+          query_service_version: ${{ steps.query-service.outputs.tag }}
+          gateway_service_version: ${{ steps.gateway-service.outputs.tag }}
+          entity_service_version: ${{ steps.entity-service.outputs.tag }}
+
+      - name: Create Pull Request
+        id: cpr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.TOKEN }}
+          commit-message: Updates helm charts
+          committer: GitHub <noreply@github.com>
+          author: ${{ github.actor }} <${{ github.actor }}@users.noreply.github.com>
+          signoff: false
+          branch: update-helm-charts
```

Comment:
What if there was a branch that exists with this name?
Thought, can we have branch-name as `update-release-branch-on-<MM:DD:YY:HH>`?

Reply:
it pushes to same branch if there's existing branch with that name. having static branch name can help us in having conditions on the basis of branch name in case of test workflow

Reply:
Is it possible to check first if that branch exists or not?

Reply:
it checks if branch exists or not and then also checks if that branch is even with main. If the branch is even with main it will delete that branch else it will push to same branch.